tls/s2n_handshake_io.c

```diff
@@ -1077,6 +1078,51 @@ static int s2n_handle_retry_state(struct s2n_connection *conn)
     return 0;
 }
 
+static int s2n_write_keylog(struct s2n_connection *conn)
+{
+    if (!conn->config->keylog_cb) {
+            return 0;
+    }
+
+    const size_t keylog_line_max_length = sizeof("CLIENT_RANDOM ") + 2 * sizeof(conn->secure.client_random) +
+        2 * sizeof(conn->secure.master_secret) + 2;
```

Comment:
Any chance this can overflow?

Reply:
There's nothing in this line which isn't a compile time constant. The compiler would catch an overflow here.